src/test/kotlin/day05/WandTest.kt

```
@@ -0,0 +1,95 @@
+        try {
+            val wandAfterNameNull = Wand(null, 5.0)
+        } catch(e: IllegalArgumentException) {
+            println("name is null")
+        }
+    }
+
+    @Test
+    fun `wandConstructorNameIsTooShort`() {
+        try {
+            val wandAfterNameNull = Wand("?", 5.0)
+        } catch(e: IllegalArgumentException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)